components/contribution-flow/graphql/queries.js

```diff
@@ -16,6 +16,16 @@ export const contributionFlowAccountWithTierQuery = gqlV2/* GraphQL */ `
     account(slug: $collectiveSlug, throwIfMissing: false) {
       ...ContributionFlowAccountFields
     }
+    loggedInAccount {
```

Comment:
We have two problems there:
1. We should also add that to `contributionFlowAccountQuery`, otherwise it will never be displayed on `/donate`
2. This will only work if the users are already logged-in, so when they arrive from a different page. Try to open `/babel/contribute/bronze-sponsors-356/checkout` directly, you won't see the string that you've added (even if you're signed-in).

I suggest moving this `loggedInAccount` part to a separate query, with a `skip` set to true if `LoggedInUser` is nullish.

Reply:
I moved it into a seperate query as below:
```diff
+  const contributionFlowLoggedInAccountQuery = gqlV2/* GraphQL */ `
+   query ContributionFlowLoggedInAccountQuery($collectiveSlug: String!) {
+     loggedInAccount {
+       memberOf(account: { slug: $collectiveSlug }) {
+        nodes {
+           id
+           totalDonations {
+             value
+           }
+         }
+      }
+     }
+  }
`;

+ const addLoggedInAccountData = graphql(contributionFlowLoggedInAccountQuery, {
+   skip: props => props.LoggedInUser === null,
+  options: props => ({
+    variables: { collectiveSlug: props.collectiveSlug },
+    context: API_V2_CONTEXT,
+  }),
+ });

- const addGraphql = compose(addAccountData, addAccountWithTierData);
+ const addGraphql = compose(addAccountData, addAccountWithTierData, addLoggedInAccountData);
```

But this broke the functionality, getting an error page because of the absence of `data.account`.
https://github.com/opencollective/opencollective-frontend/blob/110f71067d8d48cc5e5b48fe3c6b64cd26c73d3e/pages/contribution-flow.js#L190-L195


It seemed to me that only `ContributionFlowLoggedInAccountQuery` was being used and others were skipped.